spec/models/author_spec.rb

```diff
@@ -129,4 +129,37 @@
     author.valid?
     expect(author.errors[:name]).to eq([])
   end
+
+  describe ".zodiac" do
+    it "returns the zodiac that corresponds to the birthday" do
+
+      {
+        "Aries ♈️" => [Date.parse("March 21 2002"), Date.parse("April 19 2002")],
+        "Taurus ♉️" => [Date.parse("April 20 2002"), Date.parse("May 20 2002")],
+        "Gemini ♊️" => [Date.parse("May 21 2002"), Date.parse("June 20 2002")],
+        "Cancer ♋️" => [Date.parse("June 21 2002"), Date.parse("July 22 2002")],
+        "Leo ♌️" => [Date.parse("July 23 2002"), Date.parse("August 22 2002")],
+        "Virgo ♍️" => [Date.parse("August 23 2002"), Date.parse("September 22 2002")],
+        "Libra ♎️" => [Date.parse("September 23 2002"), Date.parse("October 22 2002")],
+        "Scorpio ♏️" => [Date.parse("October 23 2002"), Date.parse("November 21 2002")],
+        "Sagittarius ♐️" => [Date.parse("November 22 2002"), Date.parse("December 21 2002")],
+        # "Capricorn ♑️" => [Date.parse("December 22 2002"), Date.parse("January 19 2002")],
+        "Aquarius ♒️" => [Date.parse("January 20 2002"), Date.parse("February 18 2002")],
+        "Pisces ♓️" => [Date.parse("February 19 2002"), Date.parse("March 20 2002")],
```

Comment:
You could save some boilerplate code here by parsing the date inside the block, then you'd just have the date string here and not need to write Date.parse 20 times